skunkworks_crow/src/main/java/org/odk/share/views/ui/settings/SettingsActivity.java

```diff
@@ -183,13 +188,22 @@ private void showPasswordDialog() {
         View dialogView = factory.inflate(R.layout.dialog_password_til, null);
         TextInputLayout tlPassword = dialogView.findViewById(R.id.et_password_layout);
         tlPassword.getEditText().setText(prefs.getString(PreferenceKeys.KEY_HOTSPOT_PASSWORD, getString(R.string.default_hotspot_password)));
-
         builder.setTitle(getString(R.string.title_hotspot_password));
         builder.setView(dialogView);
+
         builder.setPositiveButton(getString(R.string.ok), (dialog, which) -> {
             String password = tlPassword.getEditText().getText().toString();
-            prefs.edit().putString(PreferenceKeys.KEY_HOTSPOT_PASSWORD, password).apply();
+
+            if(password.length()<8)
```

Comment:
@devanshi7799 the issue is disabling the `ok` button when it is less than 8 but you giving the toast

Reply:
I tried making it work like something similar to gif shown in the issue section. If the password is less than 8 characters then by ok button the original password is not changing.
Can you tell me where to make changes and where am i going wrong? Thanks :)